internal/embed/templates/global.tmpl

```diff
@@ -321,6 +321,18 @@ function reject_rpki_invalid() {
   {{ end }}
 }
 
+function force_rpki_valid() {
+  {{ if .RPKIEnable }}
+  if (net.type = NET_IP4) then {
+    if (roa_check(rpki6, net, bgp_path.last_nonaggregated) != ROA_VALID) then _reject("RPKI invalid");
```

Comment:
Let's replace the reject argument with something that clarifies that the route might not be [NotFound](https://datatracker.ietf.org/doc/html/rfc6811#section-2). Maybe something as simple as `_reject("RPKI != ROA_VALID")`?